apis/apiextensions/v1/composition_types.go

```diff
@@ -298,6 +387,35 @@ func (c *Patch) filterPatch(only ...PatchType) bool {
 	return true
 }
 
+// applyTransforms applies a list of transforms to a patch value.
+func (c *Patch) applyTransforms(input interface{}) (interface{}, error) {
```

Comment:
Moved transforms and the actual patch application into separate functions as these are used across multiple patch types